pkg/artifactory/resource_artifactory_access_token.go

```diff
@@ -405,3 +406,7 @@ func getDate(d *schema.ResourceData) (time.Time, int, error) {
 	}
 	return endDate, differenceInSeconds, nil
 }
+
+func tokenOptsToValues(t AccessTokenOptions) (url.Values, error) {
+	return query.Values(t)
```

Comment:
Why the 1-line function? It saves you no code

Reply:
The idea is to abstract out the conversion from AccessTokenOptions -> url.Values, and add a test to guarantee what kind of behavior we want (ie. "a 0 expiry in AccessTokenOptions should translate to a expires_in of 0 in the values").

We could use `query.Values` to do that conversion or anything else, but the interface/guarantee of `tokenOptsToValues` stays the same.